scheduler/src/cook/rest/api.clj

```diff
@@ -1994,10 +1994,10 @@
                                distinct
                                (remove #(contains? group-uuids %))
                                (map make-default-group))
-          groups (into (vec implicit-groups) groups)
+          db (d/db conn)
+          groups (filter (fn [group] (not (group-exists? db (:uuid group)))) (into (vec implicit-groups) groups))
```

Comment:
Maybe a comment here explaining the "why?" of this would be helpful?

Reply:
added